ipatests/test_webui/test_subid.py

```diff
@@ -142,11 +142,6 @@ def test_subid_range_deletion_not_allowed(self):
         """
         self.init_app()
         self.navigate_to_entity('subid', facet='search')
-        self.facet_button_click('add')
-        self.select_combobox('ipaowner', 'admin')
-        self.dialog_button_click('add')
-        self.wait(0.3)
-        self.assert_no_error_dialog()
```

Comment:
I believe this doesn't properly test that there is no "Remove" button.

When the Webui was still providing this button, the test for the subid remove functionality was the following (see commit [46dc1c7](https://pagure.io/freeipa/c/46dc1c76ef9996341b89167c6a0c1f63e44f4381)):
```
        self.init_app()
        self.navigate_to_entity('subid', facet='search')
        user_uid = self.get_record_pkey("some-user", "ipaowner",
                                        table_name="ipauniqueid")
        before_count = self.get_rows()
        self.delete_record(user_uid, table_name="ipauniqueid")
        after_count = self.get_rows()
        assert len(before_count) > len(after_count)
```
I think you need to select a record to properly ensure there is no "remove" command.